core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -275,6 +296,14 @@ class LoadBalancerService(
     private def generateHash(namespace: EntityName, action: ExecutableWhiskAction): Int = {
         (namespace.asString.hashCode() ^ action.fullyQualifiedName(false).asString.hashCode()).abs
     }
+
+    /** Returns a Map of invoker instance -> invoker state */
+    override def getHealthResponse(): Future[JsObject] = {
+        val res = allInvokers.map(_.map {
+            case (instance, state) => s"invoker${instance.toInt}" -> state.asString
+        }.toMap.toJson.asJsObject)
+        res
```

Comment:
There is no need to store the intermediary result:

```scala
override def getHealthResponse(): Future[JsObject] = allInvokers.map(_.map {
        case (instance, state) => s"invoker${instance.toInt}" -> state.asString
    }.toMap.toJson.asJsObject)
```

Also please refrain from using java style getter names like `getHealthResponse`. I suggest `LoadBalancer.healthStatus()`.